### ISSUE TITLE
Deprecated?

### ISSUE THREAD

**Issue by gasolinewaltz** [matthewtoast/generator-lib#8]
Why is this generator deprecated? I still use it to start new library projects...

**Comment by matthewtoast** [comment]
Hello!

I marked this as deprecated because it is based on practices that would say are outdated if not obsolete. I would strongly recommend using a Browserify or Webpack-based tool-chain instead of this generator. (I might even caution against using scaffold generators in the first place, given how advanced JS build tooling has become. Two years is a loooong time in the JavaScript ecosystem.)

I don't plan to push any further updates to this project, but if you were interested in continuing to maintain it, we could update the README to direct users to your own fork, or something like that, that way it's not just languishing here in my account?

**Comment by gasolinewaltz** [comment]
Agreed, the past two years has been crazy in the javascript world. 

So as far as the caution towards using scaffolding tools, I use them just to get the directories, files and test suites in place.

This generator is a little outdated, but it would be super easy to bring it into the present. The reason I opened this issue is because I just used it to start a new tiny library I'm working on and immediadely started editing the grunt file for my needs. 

My ideas would be as follows:
1. get rid of _head.js and _tail.js
2. configure the gruntfile to use browserify
3. configure the gruntfile to compile all js with babel to allow for es6
4. add a feature like the backbone generator has so you could enter the following command with yeoman: `yo lib:module` or something to that effect that would create a new module in the src folder.

Anyways, if you're done with it, I understand. I'd be totally down to work on all of the above and write some tests etc.

**Comment by matthewtoast** [comment]
Those ideas sound great. Since you would be taking the helm of that - how about just put it in your own repo (credit and all that), and then when you're ready, let me know, and I'll add a big message at the top of this repo that tells people to use your generator instead?
